src/main/webapp/components/BaseFormView.jsx

```diff
@@ -13,6 +14,32 @@ import { axiosCallWrapper } from '../util/axiosCallWrapper';
 import TableContext from '../context/TableContext';
 import { parseErrorMsg } from '../util/messageUtil';
 
+const CollapsiblePanelWrapper = styled(CollapsiblePanel)`
```

Comment:
Can we put this component in a separate file?

Reply:
@mchavda-splunk As discussed in a call, it is not required to create a new file for this styled component.
And this styled component is not used by any other component, hence for now it is not required to create a new file for this.